packages/content-api/README.md

```diff
@@ -0,0 +1,117 @@
+# Content Api
+
+## Install
+
+`npm install @tryghost/content-api --save`
+
+or
+
+`yarn add @tryghost/content-api`
+
+
+## Usage
+
+
+```javascript
+const GhostContentApi = require('@tryghost/content-api');
+
+const api = GhostContentApi.create({
+  host: 'https://host.tld/path/',
+  version: 'v2',
+  key: 'some-long-hex-string'
+});
+
+// Browsing posts returns Promise([Post...]);
+// The resolved array will have a meta property
+api.posts.browse({limit: 2, include: 'tags,authors'});
+api.posts.browse();
+
+// Reading posts returns Promise(Post);
+api.posts.read({slug: 'something'});
+api.posts.read({slug: 'something'}, {formats: ['html', 'plaintext']});
```

Comment:
Pages are missing.
Posts and Pages live on different routes on content api v2.

Reply:
Added to the README!